docs/features/opencdc-record.mdx

```diff
@@ -0,0 +1,132 @@
+            "float64": 1.2,
+            "int": 1,
+            "int32": 1,
+            "int64": 1,
+            "string": "orange"
+        }
+    }
+}
+```
+
+:::note
+`.Position`, `.Key`, and `.Payload.Before` are represented as `Base64` encoded in the example above because these will be a byte slice when represented as JSON.
+:::
+
+## Metadata constants
+
+As part of an OpenCDC record, there will be a set of constants provided for some metadata fields that will vary depending on the connector. These constants can be common to all **OpenCDC** records as part of our standard, some related to our own **Conduit** plugins, and others that will be provided by the **Connector SDK** independently. These constants can be useful to define conventions that will be then used by Conduit to expand its functionality. Here are the ones you can find:
+
+### OpenCDC
+
+- `opencdc.createdAt` can contain the time when the record was created in the 3rd party system. The expected format is a Unix timestamp in nanoseconds.
+- `opencdc.readAt` can contain the time when the record was read from the 3rd party system. The expected format is a Unix timestamp in nanoseconds.
+- `opencdc.version` contains the version of the OpenCDC format (e.g., "v1"). This field exists to ensure the OpenCDC format version can be easily identified in case the record gets marshaled into a different untyped format (e.g. JSON).
+
+```json
+{
+    ...
+    "metadata": "{
+        "opencdc.createdAt": "1663858188836816000",
+        "opencdc.readAt": "1663858188836816000",
+        "opencdc.version": "v1",
+        ...
+    }",
+    ...
+}
+```
+
+### Conduit
+
+Only available in records once they are read by a source plugin:
+
+- `conduit.source.plugin.name` is the name of the source plugin that created the record.
+- `conduit.source.plugin.version` is the version of the source plugin that created the record.
+
+```json
+{
+    ...
+    "metadata": "{
+        "conduit.source.plugin.name": "example",
+        "conduit.source.plugin.version": "v1",
+        ...
+    }",
+    ...
+}
+```
+
+Only available in records once they are written by a destination plugin:
+
+- `conduit.destination.plugin.name` is the name of the destination plugin that has written the record.
+- `conduit.destination.plugin.version` is the version of the destination plugin that has written the record.
+
+```json
+{
+    ...
+    "metadata": "{
+        "conduit.destination.plugin.name": "example",
+        "conduit.destination.plugin.version": "v1",
+        ...
+    }",
+    ...
+}
+```
+
+### Connector SDK
+
+These metadata fields will depend on the Connector SDK, but its contants will use the same namespace format as above by convention, indicating first the connector name that are providing them.
+
+Using the same [example above](#representation), you'll find there is a metadata key named `file.path`, and this indicates a metadata constant named `file.path` provided by the `file` builtin plugin.
```

Comment:
Scratch builtin, it could be a standalone plugin. Also I wouldn't call this a constant, it's just a value that gets added by the connector to metadata, who knows if it's a constant or not.

```suggestion
Using the same [example above](#representation), you'll find there is a metadata key named `file.path`, which indicates a metadata value provided by the `file` plugin.
```